website/views/shared/profiles/achievements.jade

```diff
@@ -213,7 +212,7 @@ div(ng-if='::profile.achievements.costumeContest')
   .achievement.achievement-costumeContest
   h5=env.t('costumeContest')
   small
-  =env.t('costumeContestText')
+    =env.t('costumeContestText')
```

Comment:
I changed that to conform to the indentations of the surrounding blocks of similar style: [these](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/shared/profiles/achievements.jade#L198-210) and [these](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/shared/profiles/achievements.jade#L219-231)